### DISCUSSION TITLE
explore postgres-wasm

### DISCUSSION THREAD

**Discussion by Newbie012** [ts-safeql/safeql#109] [Ideas]
https://github.com/snaplet/postgres-wasm - This could drastically simplify the migrations option and won't require developers to install Postgres.

**Comment by karlhorky** [comment]
This sounds really cool too!

Would this remove the need for `libpg-query` and then allow installation on Windows? (right now, not possible to use SafeQL on Windows machines: https://github.com/ts-safeql/safeql/issues/80)

**Comment by Newbie012** [comment]
@karlhorky sadly no.

---

Placing it here for reference - https://github.com/leinelissen/embedded-postgres
